tests/test_pydelfi.py

```diff
@@ -113,21 +113,6 @@ def simulator(params):
     )
     assert samples.shape[1] == len(theta0)
 
-    # TARP not yet available with pydelfi
-    metric = PosteriorCoverage(
-        out_dir=Path('./toy_pydelfi'), num_samples=2,
-        sample_method='emcee', labels=[f'$\\theta_{i}$' for i in range(3)],
-        plot_list=["tarp"],
-        sample_params={'num_chains': 6},
-    )
-    unittest.TestCase().assertRaises(
```

Comment:
I agree that we want to remove the `assertRaises` part of this unit test, but we should still call `metric(...)` to check that running TARP works

Reply:
Added, with also the other PosteriorCoverage metrics